content/php/concepts/math-functions/terms/tan/tan.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'tan()'
+Description: 'Returns the tangent of the angle passed as the argument.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`tan()`** function  returns the tangent of the angle passed as the argument. The return value is a floating-point number.
+
+## Syntax
+
+```pseudo
+$result = tan($angle);
+```
+
+Where `$angle` is the angle in radians.
+
+## Codebyte Example
+
+Using `tan()` to return the tangent of the angle of `0.8`:
+
+```codebyte/php
+<?php
+$angle = 0.8; // Angle in radians
+$tangent = tan($angle);
+
+echo "Tangent of $angle radians: $tangent\n";
+?>
+
+```
+## Output
```

Comment:
You keep marking this resolved. Are you misunderstanding me? Codebytes should not be followed by an Output section. I want you to delete the Output header and the code block that follows it. I.e. I want you to remove the text entirely from the PR.

Reply:
@SSwiniarski now check i have solved